pkgs/os-specific/linux/rtl8821ce/default.nix

```diff
@@ -7,13 +7,13 @@
 
 stdenv.mkDerivation rec {
   pname = "rtl8821ce";
-  version = "${kernel.version}-unstable-2024-01-20";
+  version = "${kernel.version}-unstable-2024-03-26";
```

Comment:
If we are relying on unstable edition all the time, it makes sense to add `unstableGitUpdater` so that ryntm bot ends up auto updating this.

Reply:
Nevermind, unstableGitUpdater may break the versioning scheme.